src/main/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImpl.java

```diff
@@ -0,0 +1,789 @@
+        LOG.info("processACHBatchDetails: Files loaded with one or more failed rows: " + numPartial);
+        if (!partialProcessingSummary.isEmpty()) {
+            for (String failingFileName : partialProcessingSummary.keySet()) {
+                ArrayList<String> errorsEncountered = (ArrayList)partialProcessingSummary.get(failingFileName);
+                LOG.info("processACHBatchDetails: ACH input file contained "+ errorsEncountered.size() + " rows that could not be processed.");
+                for (Iterator iterator = errorsEncountered.iterator(); iterator.hasNext();) {
+                    String dataError = (String) iterator.next();
+                    LOG.info("processACHBatchDetails: " + dataError);
+                }
+            }
+        }
+        LOG.info("processACHBatchDetails: Files with errors: " + numFail);
+        LOG.info("processACHBatchDetails: ==== End Summary ====");
+        
+        // For now, return true even if files or rows did not load successfully. Functionals will address the failed rows/files accordingly.
+        return true;
+    }
+
+    /**
+     * Create a collection of the files to process with the mapped value of the BatchInputFileType
+     */
+    // This is a slightly-tweaked copy of a method from CustomerLoadServiceImpl.
+    protected Map<String,BatchInputFileType> getListOfFilesToProcess() {
+        Map<String,BatchInputFileType> inputFileTypeMap = new LinkedHashMap<String, BatchInputFileType>();
+
+        for (BatchInputFileType batchInputFileType : batchInputFileTypes) {
+
+            List<String> inputFileNames = batchInputFileService.listInputFileNamesWithDoneFile(batchInputFileType);
+            if (inputFileNames == null) {
+                criticalError("BatchInputFileService.listInputFileNamesWithDoneFile(" + batchInputFileType.getFileTypeIdentifer()
+                        + ") returned NULL which should never happen.");
+            } else {
+                // update the file name mapping
+                for (String inputFileName : inputFileNames) {
+
+                    // filenames returned should never be blank/empty/null
+                    if (StringUtils.isBlank(inputFileName)) {
+                        criticalError("One of the file names returned as ready to process [" + inputFileName
+                                + "] was blank.  This should not happen, so throwing an error to investigate.");
+                    }
+
+                    inputFileTypeMap.put(inputFileName, batchInputFileType);
+                }
+            }
+        }
+
+        return inputFileTypeMap;
+    }
+
+    /**
+     * Processes a single ACH input file.
+     */
+    // Portions of this method are based on the code and logic from CustomerLoadServiceImpl.loadFile
+    protected List<String> loadACHBatchDetailFile(String inputFileName, BatchInputFileType batchInputFileType) {
+        List<String>failedRowsErrors = new ArrayList<String>();

```

Comment:
Not a huge deal, but technically, you don't need the String between the braces after new ArrayList.